packages/sdk/explorer/src/base.ts

```diff
@@ -14,7 +14,7 @@ export interface ContractMapping {
   fnMapping: Map<string, ABIDefinition>
 }
 
-export const getContractDetailsFromContract = async (
+export const getContractDetailsFromContract: any = async (
```

Comment:
why any?  it looks like the type is easy to infer?

Reply:
ts is upset without `any`. This might be due to a difference in types of the `ContractDetails` interface.

Reason for not updating the types is to limit the changes to those required to transition to node18.

Reply:
😢